bundle/edu.gemini.itc.web/src/main/resources/servlets/web/ITCgmos.html

```diff
@@ -118,162 +118,104 @@
 
 				<i>Choose one SED, the redshift and extinction</i></td>
 			</tr>
+
 			<tr>
 				<td><input value="LIBRARY_NON_STAR" name="Distribution" type="radio" /></td>
-				<td colspan="2">Library spectrum of a non-stellar object <select name="nsSpectrumType" size="1">
-				<option value="elliptical-galaxy">
-				elliptical galaxy</option>
-
-				<option selected="selected" value="spiral-galaxy">
-				spiral galaxy (Sc)</option>
-				<!--<option value="QSO">QSO</option>-->
-                <option value="QSO">QSO (80-855nm)</option>
-                <option value="QSO2">QSO (276-3520nm)</option>
-
-                <option value="Orion-nebula">
-				HII region (Orion)</option>
-                <option value="Planetary-nebula">Planetary nebula (NGC7009: 100-1100nm)</option>
-                <option value="Planetary-nebula2">Planetary nebula (IC5117: 480-2500nm)</option>
-
-                </select></td>
+				<td colspan="2">Library spectrum of a non-stellar object
+					<select name="nsSpectrumType" size="1">
+						<option value="Mars">Mars (540 nm - 5.0 µm)</option>
+						<option value="Jupiter">Jupiter (530 nm - 5.9 µm)</option>
+						<option value="Saturn">Saturn (530 nm - 5.9 µm)</option>
+						<option value="Uranus">Uranus (540 nm - 5.0 µm)</option>
+						<option value="Neptune">Neptune (540 nm - 5.0 µm)</option>
+						<option value="none" disabled></option>
+						<option value="Orion-nebula">HII region (Orion, 100 nm - 1.1 µm)</option>
+						<option value="Planetary-nebula">Planetary nebula (NGC 7009, 100 nm - 1.1 µm)</option>
+						<option value="Planetary-nebula2">Planetary nebula (IC 5117, 480 nm - 2.5 µm)</option>
+						<option value="none" disabled></option>
+						<option value="elliptical-galaxy">Elliptical galaxy (22 nm - 9.7 µm)</option>
+						<option value="spiral-galaxy" selected>Spiral galaxy (Sc, 22 nm - 9.7 µm)</option>
+						<option value="none" disabled></option>
+						<option value="QSO">QSO (80 nm - 0.855 µm)</option>
+						<option value="QSO2">QSO (276 nm - 3.52 µm)</option>
```

Comment:
Again, here there seem to be new values. Do we need to handle them in the ITC servlet?